app/controllers/log_controller.rb

```diff
@@ -7,11 +7,10 @@ def create
     @entry = LogEntry.new entry_params.merge(user: @current_user)
     if @entry.save
       flash[:success] = 'Log entry was successfully created.'
-      redirect_to log_index_path
     else
       flash[:danger] = @entry.errors.full_messages
-      render :index
     end
+    redirect_to log_index_path
```

Comment:
Yeah generally we render on failure because redirecting causes the user to lose all of the form data they may have filled out.
However, in this case, there's only one thing that can possibly be wrong and if it is, it means you haven't filled anything out. So carry on.